src/core/Header/HeaderLinks.tsx

```diff
@@ -29,23 +28,60 @@ export const HeaderLinks: React.FC<
   searchButton,
   className,
 }) => {
-  const signedIn = sessionState.signedIn;
+  const { signedIn, logOut } = sessionState;
+  const formRef = useRef<HTMLFormElement>(null);
+
   const headerLinkClasses =
     "ui-text-menu2 md:ui-text-menu3 !font-bold py-16 text-neutral-1300 dark:text-neutral-000 md:text-neutral-1000 dark:md:text-neutral-300 hover:text-neutral-1300 dark:hover:text-neutral-000 active:text-neutral-1300 dark:active:text-neutral-000 transition-colors";
 
+  const dropdownMenuLinkClasses =
+    "block p-8 ui-text-menu3 font-semibold text-neutral-1000 dark:text-neutral-300 hover:bg-neutral-100 dark:hover:bg-neutral-1200 active:bg-neutral-200 dark:active:bg-neutral-1100";
+
+  const onClickLogout = (e: MouseEvent) => {
+    e.preventDefault();
+    formRef.current?.submit();
+  };
+
+  const DashboardLink = ({ className }: { className: string }) => (
+    <a href="/dashboard" className={className}>
+      Dashboard
+    </a>
+  );
+
+  const LOGOUT_FORM = (
+    <form ref={formRef} method="post" action={logOut.href} className="hidden">
```

Comment:
This looks about right, but will be difficult to test in Storybook alone. We should test this as upon merging and releasing this will have a real impact on `docs` when docs bumps ably-ui.  Have you based this on another implementation elsewhere in ably code?

Also, the naming of this is styled like a constant when it should be more like an element, camelcase it.

Reply:
@jamiehenson yes, based on the existing Meganav. Plan to pre-release it first to test it on docs.